sirepo/job_api.py

```diff
@@ -283,6 +283,7 @@ def _validate_and_add_sbatch_fields(request_content, compute_model):
         assert m.sbatchQueue in sirepo.job.NERSC_QUEUES, \
             f'sbatchQueue={m.sbatchQueue} not in NERSC_QUEUES={sirepo.job.NERSC_QUEUES}'
         c.sbatchQueue = m.sbatchQueue
+        c.sbatchRepo = m.sbatchRepo
```

Comment:
NERSC calls these things projects (canonically) and repos (parenthetically) so let's call them projects. It'll be a bit less confusing in general, because repos mean something specific to us.

Reply:
https://github.com/radiasoft/sirepo/pull/2528/commits/ca5b244468a0e5f89041b56fb49a4eac225a8a87